storage/watcher_test.go

```diff
@@ -27,36 +30,56 @@ func TestWatcherWatchID(t *testing.T) {
 
 	idm := make(map[int64]struct{})
 
+	testKey, testValue := []byte("foo"), []byte("bar")
+
 	// synced watchings
 	for i := 0; i < 10; i++ {
-		id, cancel := w.Watch([]byte("foo"), false, 0)
+
+		id, cancel := w.Watch(testKey, false, 0)
 		if _, ok := idm[id]; ok {
 			t.Errorf("#%d: id %d exists", i, id)
 		}
 		idm[id] = struct{}{}
 
-		s.Put([]byte("foo"), []byte("bar"))
+		s.Put(testKey, testValue)
 
-		ev := <-w.Chan()
-		if ev.WatchID != id {
-			t.Errorf("#%d: watch id in event = %d, want %d", i, ev.WatchID, id)
+		evs := <-w.Chan()
+		for j, ev := range evs {
+			if !bytes.Equal(ev.Kv.Key, testKey) {
+				t.Errorf("#%d: Key got = %s, want = %s", j, ev.Kv.Key, testKey)
```

Comment:
please remove the check of key and value. This is a watchID test. We ONLY want to check watchID. We want to keep test only care about one thing each time.